src/plans/components/ProviderSelect.tsx

```diff
@@ -0,0 +1,95 @@
+import { ComponentProps, FC, ReactNode, useMemo } from 'react';
+
+import Select from '@components/common/MtvSelect';
+import { EmptyState } from '@components/EmptyState';
+import { ProviderModelGroupVersionKind, ProviderModelRef, V1beta1Provider } from '@kubev2v/types';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  EmptyStateVariant,
+  Flex,
+  FlexItem,
+  Icon,
+  SelectOption,
+  Title,
+} from '@patternfly/react-core';
+import { isEmpty } from '@utils/helpers';
+import { getResourceUrl } from 'src/modules/Providers/utils/helpers/getResourceUrl';
+import { ExternalLink } from '@components/common/ExternalLink/ExternalLink';
+import { useForkliftTranslation } from '@utils/i18n';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+type ProviderSelectProps = Pick<ComponentProps<typeof Select>, 'onSelect' | 'status'> & {
+  id: string;
+  value: string;
+  namespace: string | undefined;
+  placeholder?: string;
+  emptyState?: ReactNode;
+};
+
+export const ProviderSelect: FC<ProviderSelectProps> = ({
+  id,
+  value,
+  namespace,
+  placeholder,
+  emptyState,
+  status,
+  onSelect,
+}) => {
+  const { t } = useForkliftTranslation();
+  const [providers, providersLoaded, providerError] = useK8sWatchResource<V1beta1Provider[]>({
+    namespace,
+    groupVersionKind: { ...ProviderModelGroupVersionKind, kind: 'what' },
```

Comment:
what is kind what?

Reply:
This was added in to demo to error screenshot, will remove.